### DISCUSSION TITLE
Animation Styles and Rigs

### DISCUSSION THREAD

**Discussion by ClockTheSenior** [Sam54123/mc-world-export#56] [Ideas]
Hey, I saw that in the Issues you and TDC wanted to collaborate and use the MCPrep rigs. If you do that, please add an option to use the original rigs you made. Some people like me, really want simple rigs. Also if it is possible, maybe use Actions to change the animation style? Such as using actions for breaking, walking, swimming, placing, etc. to use custom animations and be able to tweak them too. That would allow huge customization possibilities to the user. Also, maybe add a preset for the "Mojang Promotional Trailer" animations from the Fresh Animations OptiFine pack?

Thanks a lot in advance!

**Comment by Igrium** [comment]
I never actually made any "original rigs"; that was planned for the future. The rigs (if they even count as rigs) that it uses now are procedurally generated from the Minecraft game, and the option to use those will remain.

As for animation style changes, it's highly unlikely that there will be premade "custom" animations shipping with this addon any time soon. The whole premise of the addon is to export Minecraft gameplay as-is so you can modify it yourself in Blender. However, in most cases, any mods which modify animations will be used during export.
